app/src/main/res/values/strings.xml

```diff
@@ -154,6 +154,11 @@
 
     <string name="message_couldNotOpenUrl">Couldn\'t open this URL</string>
 
+    <string name="message_disableTouch_inputDisabled">Touch input disabled</string>
+    <string name="message_disableTouch_inputEnabled">Touch input enabled</string>
+    <string name="d_disableTouch_changeKey_title">Disable touch input button</string>
+    <string name="d_disableTouch_currentCode">Current key code:</string>
```

Comment:
Could you add a simple hint that a user has to press the desired button to change the setting?

Reply:
Added. Wording may be changed though.